.github/workflows/macOS_gcc10.yml

```diff
@@ -64,6 +64,9 @@ jobs:
         brew postinstall dbus
         brew services restart dbus
         cmake -E make_directory ${{runner.workspace}}/build
+        export JAVA_HOME=$(/usr/libexec/java_home -v 16)
```

Comment:
Please check if this is really required and add remarks to jni/README.md

Reply:
AFAIK this should normally be done during the installation of the JDK. How exactly it happens depends on your OS and distro. I think, Homebrew normally doesn't set such environment variables automatically and instead prints a message for the user that this might be needed. But when using e.g. Debian the `JAVA_HOME` variable would probably be set by `apt` during the installation process.

If we add anything to the README it should only be along the lines of "You need a working JDK installation for Java version X or higher" (not sure what we require).

Reply:
> If we add anything to the README it should only be along the lines of "You need a working JDK installation for Java version X or higher" (not sure what we require).

I agree. (minimum required Java version is 9)

Reply:
The problem is that we do not only require "a working JDK installation" but we require more. Compiling JNI plugins is far away from what typically is done with a JDK.

> But when using e.g. Debian the JAVA_HOME variable would probably be set by apt during the installation process.

No, it is not: in Debian `java(c)` is in PATH but this is not enough for our needs. Usually JAVA_HOME is not needed in Debian but in our case it might be. If it is needed, we should mention it.

Reply:
`JAVA_HOME` normally shouldn't be needed. Gradle and all the standard Java tools will detect a Java installation, if it is located in one of the standard places. However, if you have multiple JDKs installed or used a non-standard installer (e.g. Homebrew) setting `JAVA_HOME` will tell the tools where to look for the JDK. This is entirely a standard Java development thing.

The Github Actions macOS image contains multiple JDKs and we installed via Homebrew (which results in a non-standard path) so setting `JAVA_HOME` is required to ensure the correct JDK is used.

If we want to be on the really safe side, we could state in the README: "A working JDK installation for Java 9 or higher is required. You may need to set `JAVA_HOME` to point to the correct JDK install".

Reply:
> The Github Actions macOS image contains multiple JDKs

Yes I can confirm that this was the reason. Otherwise a different/wrong JDK is used. This is specific to Github Actions and thus is only in this job's config and not in CMake.

For a normal installation (on my machine) where only one JDK is present, this is not needed at all.